components/window/position.md

```diff
@@ -12,42 +12,14 @@ position: 2
 
 The Window offers several ways to control its position:
 
-* [`Centered` parameter](#centered)
 * [`ContainmentSelector` parameter](#containmentselector)
 * [`Top` and `Left` parameters](#top-and-left)
+* [Centered Window](#center)
+* [Positions Example](#example)
```

Comment:
These two bullet items don't correspond to the language of the introductory sentence ("ways to control the Window position"). Either modify the bullet items, or the introduction.